candidate.cfg

```diff
@@ -6,3 +6,4 @@ accession_factory = encoded.server_defaults.enc_accession
 file_upload_bucket = encode-files
 blob_bucket = encoded-blobs
 region_search_instance = region-search.production.encodedcc.org:9201
+external_aws_s3_transfer_allow = true
```

Comment:
The functionality is OFF for all build types except "bin/deploy --candidate" builds.  Candidate is only used for test and prod at the moment but we'll need to add it to RC.  Our deploy config organization could be improved given our dev, integration, and production deployment cases.  See ENCD-4133 if interested.

Changing non-candidate builds after deploy...
1.  Update the src/encoded/production.ini to external_aws_s3_transfer_allow=true and external_aws_s3_transfer_buckets=src/encoded/.aws/direct-external-s3-list
2. Add the src/encoded/.aws/direct-external-s3-list